ingest/validation/validate_metadata.py

```diff
@@ -726,10 +727,23 @@ def validate_collected_ontology_data(metadata, convention):
     for entry in metadata.ontology.keys():
         # provision to skip validation of non-EBI OLS metadata for now
         if entry == 'organ_region':
-            continue
+            MBA_url = 'https://raw.githubusercontent.com/broadinstitute/scp-ingest-pipeline/jlc_validate_organ_region/schema/organ_region/mouse_brain_atlas/MouseBrainAtlas.csv'
```

Comment:
I think that we probably want a better place to store this - loading from a GitHub URL on a branch will depend on that branch always existing.  Also, if this is in the same repository, why are you loading it remotely, and not from its local path?

Reply:
validate_collected_ontology_data gets called from both test_validate_metadata.py and from manage_study.py. When called in those places, relative paths don't work.

I'm planning to update the URL to point to a particular commit which will slightly address the URL issue for now. Devon is helping me look into how we might use the local copy and not need to load the file from GitHub. We would like to make this more robust and not depend on an external source but didn't want that refinement to block getting the basic functionality into place.

Reply:
`bf48ed8` updated to point to first commit where file was added to repo